src/Assignment.sql

```diff
@@ -0,0 +1,1259 @@
+84) Find all customers who are not located in SanJose and whose rating is above 200.
+Query:- select CNAME , CITY , RATING from Customers where not  CITY = 'SanJose' and RATING>200;
+Output:-
+        +-------+--------+--------+
+        | CNAME | CITY   | RATING |
+        +-------+--------+--------+
+        | Grass | Berlin |    300 |
+        +-------+--------+--------+
+------------------------------------------------------------------------------------------------------------------------
+85) Give a simpler way to write this query.SELECT snum, sname, city, comm FROM
+    salespeople WHERE (comm > + 0.12 OR comm < 0.14);
+ Query:- select * from Salespeople where (COMM>12 or COMM<14);
+ Output:-
+         +------+---------+-----------+------+
+         | SNUM | SNAME   | CITY      | COMM |
+         +------+---------+-----------+------+
+         | 1001 | Peel    | London    |   12 |
+         | 1002 | Serres  | SanJose   |   13 |
+         | 1003 | AxelRod | New York  |   10 |
+         | 1004 | Motika  | London    |   11 |
+         | 1007 | Rifkin  | Barcelona |   15 |
+         | 1008 | Fran    | London    |   25 |
+         +------+---------+-----------+------+
+------------------------------------------------------------------------------------------------------------------------
+
+86) Which salespersons attend to customers not in the city they have been assigned to?
+Query :-
+     select SNAME , COMM , CNAME , RATING from Salespeople s,  Customers c where s.SNUM = C.SNUM and s.city !=c.CITY;
+Output :-
+            +---------+------+----------+--------+
+            | SNAME   | COMM | CNAME    | RATING |
+            +---------+------+----------+--------+
+            | AxelRod |   10 | Giovanni |    200 |
+            | Serres  |   13 | Grass    |    300 |
+            | Motika  |   11 | Pereira  |    100 |
+            | Rifkin  |   15 | Cineros  |    300 |
+            +---------+------+----------+--------+
+------------------------------------------------------------------------------------------------------------------------
+87) Which salespeople get commission greater than 0.11 are serving customers rated less than 250?
+Query:- select SNAME, COMM, CNAME, RATING from Salespeople s,
+            -> Customers c where s.SNUM = c.SNUM AND (s.COMM>11 AND c.RATING<250);
+output:-
+        +--------+------+---------+--------+
+        | SNAME  | COMM | CNAME   | RATING |
+        +--------+------+---------+--------+
+        | Peel   |   12 | Hoffman |    100 |
+        | Serres |   13 | Liu     |    200 |
+        | Peel   |   12 | Clemens |    100 |
+        +--------+------+---------+--------+
+------------------------------------------------------------------------------------------------------------------------
+88) Which salespeople have been assigned to the same city but get different commission percentages?
+Query:- select SNAME, COMM, CITY  from salespeople where CITY='London';
```

Comment:
You should not put the condition check city='London' but instead of this, you should try to write a generic query that will search all the records and then return the result.